README.md

```diff
@@ -8,6 +8,8 @@
 
 It supports creating custom visualizations with your own data set and correlate metadata with topics. This site also has a Gallery of models and visualizations with data such as the Microsoft employee engagement survey, called MSPoll, and feedback on the Windows Engineering System.
 
+<a href="https://www.youtube.com/watch?v=pcsZPozC9uA"><img src="https://github.com/microsoft/LongitudinalDifferenceInDifferencesPy/blob/master/Images/IntroToBrowseCloudYouTube.PNG" /></a>
```

Comment:
Should this be underneath the header?

Reply:
I'll move it below header.

Reply:
I like the new image you gave me. I'll change that as well.